package.json

```diff
@@ -20,10 +20,11 @@
     "lint:tsc": "tsc -b",
     "lint:eslint": "eslint --cache --cache-strategy content .",
     "lint:prettier": "prettier --cache --check .",
-    "test": "vitest --project unit",
-    "e2e": "run-s -c e2e:build e2e:vitest",
+    "test": "vitest --run --project unit",
+    "e2e": "run-s -c e2e:build e2e:vitest e2e:vscode",
     "e2e:build": "npm run build",
-    "e2e:vitest": "vitest --project e2e"
+    "e2e:vitest": "vitest --run --project e2e",
+    "e2e:vscode": "vscode-test"
```

Comment:
When using tests with `vscode-test`, we manipulate VS Code using the API exported from the vscode package. Therefore, it is more accurate to call this an integration test rather than an E2E test.

However, we have intentionally named the npm script `e2e:vscode`. This is because when executing `e2e:*` npm scripts, there is a mechanism in place to build the source code beforehand. `vscode-test` does not work unless the source code is built beforehand.